modules/ui_extra_networks.py

```diff
@@ -198,10 +201,21 @@ def create_html_for_item(self, item, tabname):
             "search_term": item.get("search_term", ""),
             "metadata_button": metadata_button,
             "search_only": " search_only" if search_only else "",
+            "sort_keys": sort_keys,
         }
 
         return self.card_page.format(**args)
 
+    def get_sort_keys(self, path):
+        """
+        List of default keys used for sorting in the UI.
+        """
+        return {
+            "date_created": int(Path(path).stat().st_ctime or 0),
+            "date_modified": int(Path(path).stat().st_mtime or 0),
+            "name": Path(path).name.lower(),
+        }

```

Comment:
Please at least only do the most work here once, instead of constructing three`Path`s and doing two `stat` calls per file :grin:

```suggestion
        pth = Path(path)
        stat = pth.stat()
        return {
            "date_created": int(stat.st_ctime or 0),
            "date_modified": int(stat.st_mtime or 0),
            "name": pth.name.lower(),
        }
```